recipe/meta.yaml

```diff
@@ -1,17 +1,19 @@
 {% set name = "conda-devenv" %}
-{% set version = "1.0.3" %}
+{% set version = "1.0.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  git_url: https://github.com/ESSS/{{ name }}.git
-  git_rev: {{ version }}
+  url: https://github.com/ESSS/conda-devenv/archive/{{ version }}.tar.gz
+  sha256: a0a88f7d482ec6293aee52a3026055e32a8719ec828f0496fc9b5c084c7ab33f
 
 build:
-  number: 1
-  script: python -m pip install . --no-deps --ignore-installed
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    - python -m pip install . --no-deps --ignore-installed
```

Comment:
Oh just realized that we should update this to:

```
"{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

But we can do that the next time we update.